### ISSUE TITLE
Increased number of Icons for Calendar

### ISSUE THREAD

**Issue by markjsmith** [marksie1988/atomic-calendar-revive#166]
## Summary

I’d like to add more icons and filters to my calendar

## Motivation

I have a joint calendar that puts the name of the person in and so a filter by name is great, but then other calendars are only for one person, without the name and so i cannot use the same icon

## Describe alternatives you've considered

Another solution could be to add a filter yes/no option for each calendar?

## Additional context

I had a try at adding Icon4 and Icon5 info to the config, but I’m no coder and it didn’t work!

**Comment by marksie1988** [comment]
Hey Mark, 

Im thinking about changing this so you add an entity multiple times and have a different icon for each one. e.g.

```
entities:
  - entity: mycalendar
     icon: star  
     blacklist: bob
  - entity: mycalendar
     icon: house
     whitelist: bob
```

this would mean that you can have unlimited icons and whitelists / blacklists.

let me know what you think.

**Comment by markjsmith** [comment]
So i could set up filters on one calendar, and use the same icon on another calendar without a filter? Perfect!

**Comment by marksie1988** [comment]
Fixed in #237
